### PR TITLE
Feature add performance benchmarks

### PR DESCRIPTION
**Pull request by rawrmonstar**
This is related to the updated builder-victory-component archetype https://github.com/FormidableLabs/builder-victory-component/pull/78

Add performance benchmarks for:
- [x] VictoryArea
- [x] VictoryAxis
- [x] VictoryBar
- [x] VictoryCandlestick
- [x] VictoryChart
- [x] VictoryErrorBar
- [x] VictoryGroup
- [x] VictoryLine
- [x] VictoryScatter
- [x] VictoryStack
- [x] VictoryVoronoi
- [x] VictoryVoronoiTooltip

Perf benchmarks can be run with `npm run benchmark`.  They take awhile. If you're only interested in one component you can change the regular expression in `./perf/client/main.js` to target the file named after the component you want to test, e.g., 

For example, if you were only interested in `<VictoryArea>`
```diff
- const benchesReq = require.context(".", true, /\.bench.js?$/);
+ const benchesReq = require.context(".", true, /victory-area\.bench.js?$/);
```

### CODE REVIEW

**Comment by brandly** [comment]
Hey! Thanks for Victory!! I'm loving it so far.

Overall, I've been pretty happy with performance, but I'm glad to see there's more effort targeting performance.

The main performance issue I've had involved event handlers, `labelComponent`, and mutations, like in [the tooltip demo](https://formidable.com/open-source/victory/recipes/tooltip). That demo works great, but my app crawled to a halt with ~500 data points.

I haven't dug into the source to really understand how mutations work in their current form, but maybe this info can help inform these benchmarks. 😄 

Thanks again!

File: perf/client/bench/components/victory-area.bench.js [hunk 1]
```diff
@@ -0,0 +1,388 @@
...
+import VictoryStack from "src/components/victory-stack/victory-stack";
+import { element, suiteOpts } from "../bench-helpers.js";
+
+suite("<VictoryArea> data", () => {
+  benchmark("identity fn", () => ReactDOM.render(<VictoryArea />, element));
+
+  benchmark("10 pts", () => ReactDOM.render(
+    <VictoryArea
+      data={[
+        {x: 1, y: 1},
```
Comment (boygirl) [inline perf/client/bench/components/victory-area.bench.js]: Please extract all these data arrays into a separate file, or better yet make a couple functions to generate random data sets of a given size

File: perf/client/bench/components/victory-axis.bench.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      tickValues={[
+        "1", "2", "3", "4", "5", "6", "7", "8", "9", "10"
+      ]}
+    />, element));
+
+  benchmark("50 ticks", () => ReactDOM.render(
+    <VictoryAxis
+      label="Integers"
+      tickValues={[
+        "1", "2", "3", "4", "5", "6", "7", "8", "9", "10",
```
Comment (boygirl) [inline perf/client/bench/components/victory-axis.bench.js]: Please do the same kind of thing with `tickValues`

**Comment by boygirl** [comment]
@rawrmonstar Sorry for the delay. This is approved pending code organizational changes and release of https://github.com/FormidableLabs/builder-victory-component/pull/78

**Comment by rawrmonstar** [comment]
@boygirl I've extracted the sample data and created sample data generator helpers.  This is ready for review again 😄

File: perf/client/bench/components/victory-area/victory-area.bench.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import VictoryArea from "src/components/victory-area/victory-area";
+import VictoryChart from "src/components/victory-chart/victory-chart";
+import VictoryGroup from "src/components/victory-group/victory-group";
+import VictoryStack from "src/components/victory-stack/victory-stack";
+import { element, suiteOpts } from "../../helpers/bench-helpers.js";
+import { dataSamples, dataAccessorSamples } from "./sample-data";
+
+suite("<VictoryArea> data", () => {
```
Comment (ryan-roemer) [inline perf/client/bench/components/victory-area/victory-area.bench.js]: GLOBAL: Should we use `describe` instead of `suite`?

**Comment by ryan-roemer** [comment]
LGTM!

**Comment by boygirl** [comment]
Approved! Thanks again for this work @rawrmonstar

**Pull request merged**